dnschecker_test.go

```diff
@@ -102,7 +102,7 @@ func TestDNSChecker(t *testing.T) {
 	}
 }
 
-func TestDNSCheckerWithAgressiveTimeout(t *testing.T) {
+func TestDNSCheckerWithAggressiveTimeout(t *testing.T) {
```

Comment:
Bit confused by some of the duplicate commits like this one - I remember adding this in one of my commits. Did you pull in the changes made in the branch before adding your own changes?